ui/components/Modal.js

```diff
@@ -290,3 +291,110 @@ function Modal(props) {
 }
 
 export default React.memo(Modal);
+
+function RJSFModalWrapper({
+  handleClose,
+  schema,
+  uiSchema = {},
+  initialData = {},
+  handleSubmit,
+  title,
+  submitBtnText,
+  helpText,
+}) {
+  const formRef = useRef();
+  const classes = useStyles();
+  const formStateRef = useRef();
+  const [canNotSubmit, setCanNotSubmit] = useState(false);
+  const [snackbar, setSnackbar] = useState(false);
+  const [loadingSchema, setLoadingSchema] = useState(true);
+
+  useEffect(() => {
+    setCanNotSubmit(false);
+    const handleDesignNameCheck = () => {
+      const designName = title?.toLowerCase();
+      const forbiddenWords = ['untitled design', 'Untitled', 'lfx'];
+
+      for (const word of forbiddenWords) {
+        if (designName?.includes(word)) {
+          setSnackbar({
+            severity: 'warning',
+            message: `Design name should not contain Untitled Design, Untitled, LFX`,
```

Comment:
whats the Untitled LFX ?

Reply:
i am not sure about this, this was added by @Yashsharma1911 i guess.